src/partials/languages/it.html

```diff
@@ -23,23 +23,40 @@
 <!-- Translations: Italian -->
 {% macro t(key) %}{{ {
   "language": "it",
+  "direction": "ltr",
   "clipboard.copy": "Copia",
   "clipboard.copied": "Copiato",
   "edit.link.title": "Modifica",
   "footer.previous": "Precedente",
   "footer.next": "Prossimo",
+  "footer.title": "Piede",
+  "header.title": "Intestazione",
   "meta.comments": "Commenti",
   "meta.source": "Sorgente",
+  "nav.title": "Navigazione",
   "search.config.lang": "it",
+  "search.config.pipeline": "trimmer, stopWordFilter",
```

Comment:
Almost perfect :-) This line and the next are not necessary, as they are the same as in `en.html`. They can be removed. Otherwise, this is ready for merge!

Reply:
Done, thank you